src/main/java/com/hubspot/jinjava/util/ObjectTruthValue.java

```diff
@@ -39,7 +39,19 @@ public static boolean evaluate(Object object) {
     }
 
     if (object instanceof Number) {
-      return ((Number) object).intValue() != 0;
+      if (object instanceof Integer) {
+        return (Integer) object != 0;
+      }
+      if (object instanceof Double) {
+        return (Double) object != 0;
+      }
+      if (object instanceof Long) {
+        return (Long) object != 0;
+      }
+      if (object instanceof Short) {
+        return (Short) object != 0;
+      }
+      return (Float) object != 0;
```

Comment:
Alternatively, I could just do:
```
return ((Number)object).doubleValue() != 0;
```

Reply:
Unfortunately I don't think that will quite work - Python doesn't treat `0.0` as truthy

Reply:
Yeah, I wrote it like that thinking it would be faster, but I tested it. This shorter way is faster for longs, about the same time for floats and ints so I'm going to do the simpler approach